geode-gfsh/src/test/java/org/apache/geode/management/internal/cli/functions/WanCopyRegionFunctionTest.java

```diff
@@ -562,13 +562,13 @@ public void wanCopyRegion_verifyExceptionThrownWhenExceptionWhileSendingBatch()
   public void executeWanCopyRegionFunctionInNewThread_verifyErrorWhenAlreadyRunningCommand()
       throws BatchException70, InterruptedException, ExecutionException {
     Object[] options = new Object[] {"myRegion", "mySender", false, 1L, 1};
-    Future<CliFunctionResult> future = executeAsyncWanCopyRegionFunction(options);
+    Future<CliFunctionResult> future = executeAsyncWanCopyRegionFunction(options, 1000);
 
     // Wait for the execute function to start
-    Thread.sleep(100);
+    Thread.sleep(500L);
```

Comment:
Is there something we can wait on rather than just using a Thread.sleep?

Reply:
This PR has been deprecated.
New changes are being done here: https://github.com/apache/geode/pull/6833